QuickRadar/QRRadarSubmissionService.m

```diff
@@ -194,7 +196,81 @@ - (void)submitAsyncWithProgressBlock:(void (^)())progressBlock completionBlock:(
 			});
 		}
         
-        
+        // ------- 2-Factor-Auth Support by @steipete --------
+
+        if ([bouncePage.pageContent containsString:@"protected with two-factor authentication"]) {
```

Comment:
We'll have to keep an eye on that one if they change the strings. But I guess that's true for all of this scraping :)